.github/workflows/build-and-test.yml

```
@@ -14,22 +14,45 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+
+jobs:
+  generate-matrix:
+    name: "Generate matrix from cabal"
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Extract the tested GHC versions
+        id: set-matrix
+        uses: kleidukos/get-tested@v0.1.7.0
+        with:
+          cabal-file: get-tested.cabal
```

I missed this in the previous review, my bad! The `cabal-file` needs to point to the project's Cabal file.
```suggestion
          cabal-file: hspec-golden.cabal
```

---

Thanks for catching that, David!